src/uct/ib/rc/base/rc_ep.h

```diff
@@ -327,6 +327,15 @@ int uct_rc_fc_has_resources(uct_rc_iface_t *iface, uct_rc_fc_t *fc)
     return (fc->fc_wnd > 0) || !iface->config.fc_enabled;
 }
 
+static UCS_F_ALWAYS_INLINE void uct_rc_fc_restore_wnd(uct_rc_iface_t *iface,
+                                                      uct_rc_fc_t *fc)
+{
+    fc->fc_wnd = iface->config.fc_wnd_size;
+    if (iface->config.fc_enabled) {
```

Comment:
why need this if? Let it always be a some `MAX` value if FC is disabled

Reply:
removed